src/routes/group.route.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
+import GroupsController from '@controllers/groupe.controller';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { Routes } from '@interfaces/routes.interface';
+import validationMiddleware from '@middlewares/validation.middleware';
+
+class GroupsRoute implements Routes {
+  public path = '/groups';
+  public router = Router();
+  public groupsController = new GroupsController();
+
+  constructor() {
+    this.initializeRoutes();
+  }
+
+  private initializeRoutes() {
```

Comment:
It will be necessary to add documentation in the `swagger.yaml` of the `groups` enpoint, if you have the chance to separate the `swagger.yaml` in `users.yaml` and `groups.yaml` in this PR go ahead, if you consider that another US should be created for that add a `@Tech Debt`.